tests/bin_tests.rs

```diff
@@ -6,7 +6,27 @@ fn print_public_items() {
     let mut cmd = Command::cargo_bin("public_items").unwrap();
     cmd.arg("./tests/rustdoc_json/public_items-v0.4.0.json");
     cmd.assert()
-        .stdout(include_str!("./expected_output/public_items-v0.4.0.txt"))
```

Comment:
Just a bit of repetition here: It would be great if you could avoiding changing "expected output" (here for example). Not only to avoid conflicts with https://github.com/Enselic/public_items/pull/70 (which I will wait with merging until this PR is merged though), but also because good practice in and of itself, when doing big changes.